Assets/Amplitude/Amplitude.cs

```diff
@@ -204,6 +204,10 @@ public class Amplitude {
 		Debug.Log(message);
 	}
 
+	protected void Log<T>(string message, string property, IEnumerable<T> array) {
+		Log (string.Format("{0} {1}, {2}: [{3}]", message, property, array, string.Join(", ", array)));
```

Comment:
@EtienneMarbotic Here. :)

Reply:
The log of `array` only will display the type only, like `System.String[]`. It could be useful (it does not for me but I prefered to let it like it was if someone was using it). For me, the most important was the array content :) that's why I added the `string.Join()` method.